packages/cms-kit/src/atoms/textInput.tsx

```diff
@@ -22,23 +22,23 @@ const TextInputWrapperStyle = cssRuleWithTheme<TextInputStyleProps>(({hasError,
   alignItems: 'center',
 
   fontSize: pxToRem(FontSize.Medium),
-  fill: theme.colors.dark,
-
-  borderBottomWidth: BorderWidth.Small,
-  borderBottomStyle: 'solid',
-  borderBottomColor: hasError ? theme.colors.alert : theme.colors.gray
+  fill: theme.colors.dark
 }))
 
 const IconStyle = cssRule(() => ({
+  position: 'absolute',
   marginRight: pxToRem(Spacing.Tiny)
 }))
 
 const TextInputStyle = cssRuleWithTheme<TextInputStyleProps>(({hasError, theme}) => ({
   width: '100%',
 
+  borderBottom: `1px solid ${theme.colors.gray}`,
   transitionProperty: 'border-color',
   transitionTimingFunction: 'ease-in',
   transitionDuration: TransitionDuration.Slow,
+  // If Inupt has Icon add padding:
+  //paddingLeft: pxToRem(20),
```

Comment:
@per2plex  We need a state to dedicate if the inputfield has a icon or not, if it has an icon it should be add the `paddingLeft: pxToRem(20)`